nav2_dwb_controller/dwb_plugins/src/limited_accel_generator.cpp

```diff
@@ -68,7 +68,8 @@ void LimitedAccelGenerator::checkUseDwaParam(const nav2_util::LifecycleNode::Sha
   bool use_dwa = false;
   nh->get_parameter("use_dwa", use_dwa);
   if (!use_dwa) {
```

Comment:
Can we remove this use_dwa parameter? Not part of this PR, but if so, can we file an issue and do it soon? It seems silly to keep something that was deprecated in ROS.

Reply:
Well it does serve as a useful exception if the wrong plugins are enabled.